package-res/ccc/core/base/axis/axis.js

```diff
@@ -82,6 +89,25 @@ def('pvc.visual.Axis', pvc.visual.OptionsBase.extend({
             return this;
         },
 
+        // CDF603
+        // Function that stores something to be considered state of the axis
+        setState: function(options){
+            this._state = $.extend({}, this._state, options); 
```

Comment:
jQuery functions are only used for tipsy code.
For JS language utilities `def` is used. `def.copyOwn` can be used like:
```javascript
def.copyOwn(this._state, options);
```

Don't think that there is a need to create a copy of the current `_state` object every time it is set.